rust_src/remacs-sys/lib.rs

```diff
@@ -27,7 +27,27 @@ use libc::{c_char, c_double, c_float, c_int, c_short, c_uchar, c_void, intmax_t,
 // and alias it ourselves.
 pub type pid_t = libc::c_int;
 
-pub type Lisp_Object = EmacsInt;
+#[repr(C)]
+#[derive(Clone, Copy, PartialEq, Eq)]
+pub struct Lisp_Object(EmacsInt);
+
```

Comment:
Have you tested this on 32-bit? This has caused problems in the past: https://github.com/rust-lang/rfcs/pull/1758#issuecomment-276465077

Reply:
Sigh. No, I didn't.

That,RFC merged. In theory I can switch this to transparent now.

Reply:
I've created #568 to keep track of this.